doc/modules/reference.rst

```diff
@@ -82,6 +82,7 @@ uses.
    fetch_localizer_calculation_task
    fetch_miyawaki2008
    fetch_nyu_rest
+   fetch_nki_enhanced_surface
```

Comment:
Should we call this function "fetch_surf_nki_enhance"? Just like we have prefixed all the atlas fetching functions with fetch_atlas?